partner_compassion/views/partner_compassion_view.xml

```diff
@@ -315,21 +295,18 @@
         <field name="context">{'default_customer':1, 'search_default_customer':1, 'search_default_sponsor': 1}</field>
         <field name="filter" eval="True"/>
         <field name="help" type="html">
-            <p class="oe_view_nocontent_create">
-                Click to add a contact in your address book.
-            </p>
-            <p>
-                Odoo helps you easily track all activities related to
-                a customer: discussions, history of business opportunities,
-                documents, etc.
-            </p>
+          <p class="oe_view_nocontent_create">
```

Comment:
This change is producing your bug in Travis. It is probably due to a wrong merge during rebase.